internal/renderer/renderer.go

```diff
@@ -30,7 +30,7 @@ func Register(name string, driver driver.Driver) {
 
 // Apply applies the specified render to the specified string with the specified settings.
 // If the render is not present is the registered ones, it errors out.
-func Apply(s string, settings map[string]interface{}, renderers ...string) (string, error) {
```

Comment:
`settings` is a little bit more understandable than `ss` 😅 I guess you wanted to avoid further shadowing with `settings` package, but I think it's a little bit premature?

Reply:
yes, I don't want to shadow `settings` here

Reply:
But the package is not imported 😸

Reply:
I know, using `settings` now (will push)